rewriter/src/main/java/com/facebook/coresql/rewriter/OrderByRewriter.java

```diff
@@ -35,31 +35,29 @@
 public class OrderByRewriter
         extends Rewriter
 {
-    PatternMatcher<Set<AstNode>> matcher;
     public Set<AstNode> patternMatchedNodes;
     private static final String REWRITE_NAME = "ORDER BY without LIMIT";
 
     public OrderByRewriter()
     {
-        this.matcher = new OrderByPatternMatcher();
         this.patternMatchedNodes = new HashSet<>();
```

Comment:
Do we need this variable?

Reply:
I believe so; it's necessary in the Visitor Pattern. I don't think there is another way to know know which nodes were flagged in the pattern matching routine.